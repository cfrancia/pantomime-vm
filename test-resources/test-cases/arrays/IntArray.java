/**
 * Created by cfrancia on 26/09/16.
 */
public class IntArray {

    public static void main(String[] args) {
        int[] charArray = new int[]{1, 2, 3, 4, 5};

        for (int i = 0; i < charArray.length; i++) {
            println(charArray[i]);
        }
    }

    public static native void println(int val);

}
