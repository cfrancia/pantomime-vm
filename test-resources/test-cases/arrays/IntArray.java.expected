OUT: 1
OUT: 2
OUT: 3
OUT: 4
OUT: 5
